content/en/platform/corda/5.0-beta/deploying/deployment-tutorials/manual.md

```diff
@@ -37,69 +32,175 @@ To create the topics manually, do the following:
+  -u p2pGateway=<P2P_GATEWAY_USER> -u p2pLinkManager=<P2P_LINK_MANAGER_USER> -u rpc=<RPC_USER> \
+  connect
+```
+{{% /tab %}}
+{{% tab name="macOS" %}}
+```sh
 corda-cli.sh topic -b <BOOTSTRAP-SERVERS> -k config.properties \
-  create -r <RELICAS> -p <PARTITIONS> \
+  create -r <REPLICAS> -p <PARTITIONS> \
   -u crypto=<CRYPTO_USER> -u db=<DB_USER> -u flow=<FLOW_USER> -u membership=<MEMBERSHIP_USER> \
   -u p2pGateway=<P2P_GATEWAY_USER> -u p2pLinkManager=<P2P_LINK_MANAGER_USER> -u rpc=<RPC_USER> \
   connect
 ```
+{{% /tab %}}
+{{% tab name="Windows" %}}
+```shell
+corda-cli.cmd topic -b <BOOTSTRAP-SERVERS> -k config.properties `
+  create -r <REPLICAS> -p <PARTITIONS> `
+  -u crypto=<CRYPTO_USER> -u db=<DB_USER> -u flow=<FLOW_USER> -u membership=<MEMBERSHIP_USER> `
+  -u p2pGateway=<P2P_GATEWAY_USER> -u p2pLinkManager=<P2P_LINK_MANAGER_USER> -u rpc=<RPC_USER> `
+  connect
+```
+{{% /tab %}}
+{{< /tabs >}}
 
 ### Topic Creation by Scripting
 
-Alternatively, the Corda CLI can generate a shell script which you should review before executing against the broker.
-The shell script makes use of the `kafka-topic.sh` script provided with a Kafka installation.
+Alternatively, the Corda CLI can generate a script which you should review before executing against the broker.
+The script makes use of the `kafka-topic.sh` script provided with a Kafka installation.
 
-The Corda CLI command looks as follows:
+Run the following Corda CLI command to generate the script:
 
-```shell
+{{< tabs name="cli-script">}}
+{{% tab name="Linux" %}}
+```sh
 corda-cli.sh topic -b <BOOTSTRAP-SERVERS> -k config.properties \
-  create -r <RELICAS> -p <PARTITIONS> script -f <FILE> -c <CONCURRENCY>
+  create -r <REPLICAS> -p <PARTITIONS> script -f <FILE> -c <CONCURRENCY>
+```
+{{% /tab %}}
+{{% tab name="macOS" %}}
+```sh
+corda-cli.sh topic -b <BOOTSTRAP-SERVERS> -k config.properties \
+  create -r <REPLICAS> -p <PARTITIONS> script -f <FILE> -c <CONCURRENCY>
+```
+{{% /tab %}}
+{{% tab name="Windows" %}}
+```shell
+corda-cli.cmd topic -b <BOOTSTRAP-SERVERS> -k config.properties create -r <REPLICAS> -p <PARTITIONS> script -f <FILE> -c <CONCURRENCY>
 ```
+{{% /tab %}}
+{{< /tabs >}}
 
 Where `<FILE>` is the name of the file in which to save the script and `<CONCURRENCY>` is the number of topics to create in parallel to speed execution.
 
 For example:
 
-```shell
+{{< tabs name="cli-script-example">}}
+{{% tab name="Linux" %}}
+```sh
+corda-cli.sh topic -b kafka-1.example.com -k config.properties \
+  create -r 3 -p 10 script -f create.sh -c 6
+```
+{{% /tab %}}
+{{% tab name="macOS" %}}
+```sh
 corda-cli.sh topic -b kafka-1.example.com -k config.properties \
   create -r 3 -p 10 script -f create.sh -c 6
 ```
+{{% /tab %}}
+{{% tab name="Windows" %}}
+```shell
+corda-cli.cmd topic -b kafka-1.example.com -k config.properties create -r 3 -p 10 script -f create.cmd -c 6
+```
+{{% /tab %}}
+{{< /tabs >}}
```

Comment:
This is unfortunately a little optimistic in its use of `create.cmd` as the filename. The `create` option is hard-coded to generate a script that calls `kafka-topics.sh` and `kafka-acls.sh`. I.e. although you can run the Corda CLI on Windows to generate the file, it will always create a shell script to run on Linux/macOS/WSL.